root/static/scripts/edit/ExampleRelationships.js

```diff
@@ -10,13 +10,18 @@ MB.ExampleRelationshipsEditor = (function (ERE) {
 
 
 // Private variables
-var type0, type1, linkTypeName, linkTypeID, jsRoot;
+let type0;
+let type1;
+let linkTypeName;
+let linkTypeID;
+let jsRoot;
 
 // Private methods
 var searchUrl;
 
 // Private classes
-var RelationshipSearcher, ViewModel;
+let RelationshipSearcher;
+let ViewModel;
```

Comment:
Let's remove these declarations, and change the assignments below to `const RelationshipSearcher = ` and `const ViewModel = `.